formats/ome-converter-plugin/src/polus/plugins/formats/ome_converter/__init__.py

```diff
@@ -0,0 +1 @@
+"""Ome Converter."""
```

Comment:
There should be a field here called `__version__`.

I suspect you did not use `bumpversion` because that would have thrown an error.

Reply:
It is fixed now